packages/block-editor/src/components/iframe/index.js

```diff
@@ -260,15 +260,18 @@ function Iframe( {
 				style={ {
 					...props.style,
 					height: expand ? contentHeight : props.style?.height,
-					marginTop: scale
-						? -marginFromScaling + frameSize
-						: props.style?.marginTop,
-					marginBottom: scale
-						? -marginFromScaling + frameSize
-						: props.style?.marginBottom,
-					transform: scale
-						? `scale( ${ scale } )`
-						: props.style?.transform,
+					marginTop:
+						scale !== 1
```

Comment:
I _think_ this could be the correct intention, since `scale` has a default value of `1` even when it's not passed as a prop.